src/structure.jl

```diff
@@ -1,24 +1,32 @@
 """
-Compute the reciprocal lattice. Takes special care of 1D or 2D cases.
-We use the convention that the reciprocal lattice is the set of G vectors such
-that G ⋅ R ∈ 2π ℤ for all R in the lattice.
+Compute the inverse of the lattice. Takes special care of 1D or 2D cases.
 """
-function compute_recip_lattice(lattice::AbstractMatrix{T}) where {T}
+function compute_inverse_lattice(lattice::AbstractMatrix{T}) where {T}
     # Note: pinv pretty much does the same, but the implied SVD causes trouble
     #       with interval arithmetic and dual numbers, so we go for this version.
     n_dim = count(!iszero, eachcol(lattice))
     @assert 1 ≤ n_dim ≤ 3
     if n_dim == 3
-        2T(π) * inv(lattice')
+        inv(lattice')
```

Comment:
this is not the inverse lattice, it's the inverse transpose lattice. Don't transpose here, and transpose in compute_recip_lattice